dialog-polyfill.js

```diff
@@ -437,6 +437,10 @@
     dialog.close(returnValue);
   }, true);
 
-  window['dialogPolyfill'] = dialogPolyfill;
-  dialogPolyfill['registerDialog'] = dialogPolyfill.registerDialog;
```

Comment:
FYI, this is actually to do with the Closure compiler. Without the string export, the name is obfuscated.